model-server/src/test/kotlin/org/modelix/model/server/ReplicatedRepositoryTest.kt

```diff
@@ -327,6 +329,101 @@ class ReplicatedRepositoryTest {
         println("all successful")
     }
 
+    @RepeatedTest(value = 10)
```

Comment:
Why is it necessary to repeat this test? Overall, this test looks fairly complex. Is there no easier way to show that change propagate between different clients?